tutorials/vue/v-else-if.md

```diff
@@ -0,0 +1,57 @@
+The `v-else-if` is another directive provided by Vue.
+It must follow a `v-if` or it will not be recognized.
```

Comment:
Change this sentence to "`v-else-if` must follow a `v-if`." And move it to a separate paragraph, add an example of what happens if you put `v-else-if` without a `v-if`.